modin/experimental/cloud/rayscale.py

```diff
@@ -150,6 +150,10 @@ def _conda_requirements(self):
 
         reqs.append(self._get_modin_version())
 
+        if any(map(lambda pkg: pkg.startswith("modin"), self.add_conda_packages)):
+            # remove automatically detected modin release
+            reqs.pop()
```

Comment:
let's not invert the logic :)
append `self._get_modin_version()` if user did not override it, do not _remove_ if user has overridden